database/neo4j/neo4j.go

```diff
@@ -69,11 +71,14 @@ func (n *Neo4j) Open(url string) (database.Driver, error) {
 	authToken := neo4j.BasicAuth(uri.User.Username(), password, "")
 	uri.User = nil
 	uri.Scheme = "bolt"
+	multi := uri.Query().Get("x-multi-statement")
+	uri.RawQuery = ""
 
 	return WithInstance(&Config{
 		URL:             uri.String(),
 		AuthToken:       authToken,
 		MigrationsLabel: DefaultMigrationsLabel,
+		MultiStatement:  multi == "true",
```

Comment:
Use `strconv.ParseBool()` and handle the error properly

Reply:
Done